### DISCUSSION TITLE
Buzz app is no longer working -- Gives "Failed to execute..." Error

### DISCUSSION THREAD

**Discussion by dennisplum** [community/community#69048] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The Buzz app (https://github.com/chidiwilliams/buzz) is no longer working -- Gives "Failed to execute..." Error.

See this screenshot attached ![Screenshots of Buzz-Errors](https://github.com/community/community/assets/40610377/a72daceb-d8fe-4cc6-ad03-940ef725ccf9)

This error first appeared at the point that Buzz was going to save a Transcription File.
And now this error appears on the load of the app. I tried uninstalling and reinstalling, but with the same result.

Anyone have a solution?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @dennisplum ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by dennisplum** [comment]
Hi Lili,
Thanks for this clarification.

However, I could not find either a "Support file" nor a "Contributing
Guide" on the Buzz site. But what I did see is a "Discussion" Tab, which
appears to be focused on a discussion of various issues related to the Buzz
app -- See this screen shot https://prnt.sc/FHKyT73WfFai

 And so, I assume that moving my post to that area would make sense.
Dennis
